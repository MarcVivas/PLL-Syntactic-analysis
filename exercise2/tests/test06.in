// Assignacío de valors
a = 2;
b = -3;
// Expressio condicional
(a * -b -2 >= 0 && b < 0) ? c = a+b : c = b;
// Avaluacío d’expressions aritmetiques
a-b;
c*2;
// Expressio condicional
(c %2 == 0 && a - b < 0) ?
d = 1 :
d = -1;
