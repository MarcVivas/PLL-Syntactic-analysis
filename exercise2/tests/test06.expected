5
-6
a = 2
b = -3
c = -3
d = -1